src/main/java/ch/bfh/bti7081/model/seminar/SeminarFilter.java

```diff
@@ -0,0 +1,34 @@
+package ch.bfh.bti7081.model.seminar;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.Date;
+
+public class SeminarFilter {
```

Comment:
Ist das eine Entität, welche wir später in der DB abspeichern, oder ist das ein "Hilfs-Objekt"?

Reply:
@Azarny Ist ein Hilfsobjekt und stammt von mir..